core/tutorials/buggyamb/Docs/linux_installation_instructions.md

```diff
@@ -123,29 +123,29 @@ Now BuggyAmb is ready to restart if it crashes or if the machine is rebooted.
 
 If you are not like "enough, I am done!" yet, there are some other things to do if you want.
 
-Just like the previous "Ensuring BuggyAmb runs always" section, this one is not a requirement to get started, but...But, if you are like me you will probably want an environment as close to a real-world scenario as possible, and that port number at the hostname (:5000) will bother you: why not making requests like http://buggybits/Problem instead of http://buggybits:5000/Problem, right?
+Just like the previous "Ensuring BuggyAmb runs always" section, this one is not a requirement to get started, but...But, if you are like me you will probably want an environment as close to a real-world scenario as possible, and that port number at the hostname (:5000) will bother you: why not making requests like `http://buggybits/Problem` instead of `http://buggybits:5000/Problem`, right?
```

Comment:
```suggestion
Just like the previous "Ensuring BuggyAmb runs always" section, this one is not a requirement to get started. But, if you are like me, you will probably want an environment as close to a real-world scenario as possible, and that port number at the hostname (:5000) will bother you. Why not make requests like `http://buggybits/Problem` instead of `http://buggybits:5000/Problem`, right?
```